registry/pid.go

```diff
@@ -1,23 +1,23 @@
-package browser
+package registry
 
 import "sync"
 
-// pidRegistry keeps track of the launched browser process IDs.
-type pidRegistry struct {
+// PidRegistry keeps track of the launched browser process IDs.
+type PidRegistry struct {
```

Comment:
Yeah, I was thinking about that too. Would `registry.Pid` be better? I think for now it might be ok, considering we have things like `context.BackgroundContext`. As you've pointed out another good suggestion, we might need to refactor this again very soon, so i might leave the name as it until that next refactor 🙂